### DISCUSSION TITLE
Add Sketch does not work...

### DISCUSSION THREAD

**Discussion by claytonbp** [hlorus/CAD_Sketcher#390] [Announcements]
Hello,

I am running Cad Sketcher 0.27.2 on my iMac 8 core i9 (Vega 8GB, 72GB RAM) using Blender 3.5.1.

I have installed the add-on and PIP, however I did see a message saying QWER Master is missing.

When I select Sketcher > Add Sketch, Blender then highlights "Solverspace Select" button and that is it. Cad Sketcher does not show the axis planes.

I have watched Maker Tales video and I would expect axis planes to be display.

Is there anything I have missed?

Clayton

**Comment by hlorus** [comment]
This might be a problem with the gpu backend, see: https://github.com/hlorus/CAD_Sketcher/issues/375

**Reply by paperschool** [reply]
I am also having this issue - blender 3.5.1 - m1 max - ventura

**Reply by Buonojohn** [reply]
I am having the same issue and did not see how the message 375 solved the problem.

**Reply by Cjohnson66** [reply]
I had the same problem.
Go to Preferences > System and in GPU Backend change to OpenGL instead of the default Metal. This solved the problem for me. 

I saw this in a video but cannot remmeber which one.
